persistence/sql/migrations/sql/20241001000000000000_identity_verifiable_addresses_pk.mysql.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE identity_verifiable_addresses
```

Comment:
Isn't this missing a unique index for id?

Reply:
added!